src/layouts/divider/PDivider.vue

```diff
@@ -1,30 +1,41 @@
-<script lang="ts">
-import { getBindClass } from '@/util/functional-helpers';
+<template>
+    <div v-if="vertical" class="p-vertical-divider" />
```

Comment:
The component's class name should be the kebab case of that component name. Considering the case where you need to customize the css on the page, we set the rules like this.

```suggestion
    <component :is="vertical ? 'div' : hr" class="p-divider" :class="{vertical}" />
```